prime.rs

```diff
@@ -0,0 +1,28 @@
+use std::io;
+fn main() {
+    println!("Enter a Number: ");
+    let mut num = String::new();
+    io::stdin()
+        .read_line(&mut num)
+        .expect("Error reading number");
+    let mut n = num.trim()

```

Comment:
Remove `mut`.

`mut` is used to allow changing the value of variable. Do not use it unless required to change value dynamically in program after initialization.